ROADMAP.md

```diff
@@ -8,21 +8,25 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 See the [GitHub project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
-## Rook Ceph 1.9
+## Rook Ceph 1.10
 
-The following high level features are targeted for Rook v1.9 (early April 2022). For more detailed project tracking see the [v1.9 board](https://github.com/rook/rook/projects/24).
+The following high level features are targeted for Rook v1.10 (early August 2022). For more detailed project tracking see the [v1.10 board](https://github.com/rook/rook/projects/26).
+
+* Remove support for Ceph Octopus (support remains for Pacific and Quincy) [#10338](https://github.com/rook/rook/issues/10338)
+* Create tool to analyze cluster health and advise on resolving common health issues [Krew #32](https://github.com/rook/kubectl-rook-ceph/issues/32)
+* Check for existing subvolumes before allowing a filesystem to be uninstalled [#9915](https://github.com/rook/rook/pull/9915)
+* Use more specific cephx accounts to better differentiate the source of configuration changes [#10169](https://github.com/rook/rook/issues/10169)
+* Automate node fencing for application failover in some scenarios [#1507](https://github.com/rook/rook/issues/1507)
+* Object Store
+  * Service account authentication with the Vault agent [#9872](https://github.com/rook/rook/pull/9872)
+  * Support for AWS Server Side Encryption (SSE) [#10318](https://github.com/rook/rook/pull/10318)
 
-* Support for Ceph Quincy
-* Support for on-wire encryption and compression [#9054](https://github.com/rook/rook/issues/9054)
-* More complete set of commands for the [Rook Krew plugin](https://github.com/rook/kubectl-rook-ceph)
 
 ## Themes
 
 The general areas for improvements include the following, though may not be committed to a release.
 
 * Add alpha support for COSI (Container object storage interface) with K8s 1.24 [#7843](https://github.com/rook/rook/issues/7843)
-* iSCSI gateway deployment [#4334](https://github.com/rook/rook/issues/4334)
-* Enable the admission controller by default [#6242](https://github.com/rook/rook/issues/6242)
 * OSD encryption key rotation [#7925](https://github.com/rook/rook/issues/7925)
 * Simplify metadata backup and disaster recovery
```

Comment:
do we have any issues tracking this or is it just common in every release?

Reply:
There should be one, I need to link it...